python/dataStructure_process.py

```diff
@@ -0,0 +1,12 @@
+'''
```

Comment:
You should not mix camelcase and snake case in file naming

Reply:
Thank you, I will change that!